vars/ostreeCompose.groovy

```diff
@@ -64,7 +64,7 @@ def call(body) {
                           "export JENKINS_JOB_NAME=\"${JOB_NAME}-${current_stage}\"\n" +
                           "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
                           "export OSTREE_BRANCH=\"${OSTREE_BRANCH}\"\n"
-            pipelineUtils.rsyncResults(current_stage, 'duffy-key')
+            pipelineUtils.runTaskAndReturnLogs(current_stage, 'duffy-key')
```

Comment:
I assume you are calling this other function here but in the monolithic Jenkinsfile-rf-pipeline you are calling rsyncResults instead is that the idea?

Reply:
correct.